### PR TITLE
Send page, validate input fields as numeric

### PR DESCRIPTION
**Pull request by NovacAndrei**

### CODE REVIEW

File: src/core/i18n/translation/locales/en.ts [hunk 1]
```diff
@@ -123,7 +124,9 @@ export const translation: ITranslations = {
         Fee: {
             feeTitle: 'Transaction fee',
             gasPrice: 'Gas price',
-            gasLimit: 'Gas limit'
+            gasLimit: 'Gas limit',
+            errorGasPrice: 'Invalid Gas Price',
+            errorLimitPrice: 'Invalid Limit Price'
```
Comment (georgeburduhos) [inline src/core/i18n/translation/locales/en.ts]: Invalid Gas Limit Price should be here

File: src/screens/send/send.tsx [hunk 2]
```diff
@@ -368,6 +368,7 @@ export class SendScreenComponent extends React.Component<
     }
 
     public async alertModalAddAddress() {
+        // TODO: check this, it's not opening all the time
```
Comment (georgeburduhos) [inline src/screens/send/send.tsx]: same problem as on Delete wallet?

Comment (NovacAndrei) [reply src/screens/send/send.tsx]: Same problem, yes. The modal does not open all the time.

Comment (NovacAndrei) [reply src/screens/send/send.tsx]: I had also tried with `await delay(0)`, but it didn't help.

File: src/core/utils/format-number.ts [hunk 3]
```diff
@@ -47,3 +47,6 @@ export const formatNumber = (amount: number | BigNumber, options: INumberFormatO
         ? formattedNumber + ' ' + options.currency
         : formattedNumber;
 };
+
+const numericRegex = new RegExp(/^[+-]?\d+(\.\d+)?$/);
```
Comment (krisboit) [inline src/core/utils/format-number.ts]: is this what we want ? you should understand regex expressions you use :P
try https://regexr.com/

Comment (krisboit) [reply src/core/utils/format-number.ts]: try to understand how this regex rule works, after that i will tell you how to do it without regex :)

**Pull request merged**